common/internal_model/src/test/scala/weco/catalogue/internal_model/work/RelationsTest.scala

```diff
@@ -0,0 +1,57 @@
+package weco.catalogue.internal_model.work
+
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+import weco.catalogue.internal_model.work.generators.WorkGenerators
+
+class RelationsTest extends AnyFunSpec with Matchers {
+  it("Has zero size when empty") {
```

Comment:
nit: codebase convention is that test descriptions start with a lowercase letter, e.g.

```suggestion
  it("has zero size when empty") {
```

I don't have strong feelings either way, but let's be consistent.